modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -110,9 +110,10 @@ endif::[]
 
 == Register Clients
 
-. To register your openSUSE clients, you will need a bootstrap repository.
-Create the bootstrap repository at the command prompt, with this command:
-+
+To register your openSUSE clients, you need a bootstrap repository.
+By default, bootstrap repositories are regenerated daily.
```

Comment:
Not sure if this would be too much info, but we could also write:

```suggestion
By default, bootstrap repositories are automatically created and regenerated daily for all synced products.
```

if, this should go also to the other places.